notebooks/COS/DataDl/DataDl.ipynb

```diff
@@ -0,0 +1,851 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Fig 1.13\n",
+    "<center><img src =figures/mastp_cart2.png width =\"900\" title=\"MAST Portal Download Basket\"> </center>\n",
+    "\n",
+    "Each dataset contains *many* files, most of which are calibration files or intermediate processing files. You may or may not want some of these intermediate files in addition to the final product file.\n",
+    "In the leftmost \"Filters\" section of the Download Basket page, you can narrow which files will be downloaded (boxed in red).\n",
+    "By default, only the **minimum recommended products** (*mrp*) will be selected. In the case of most COS data, this will be the final spectrum `x1dsum` file and association `asn` file for each dataset. The mrp files for the first dataset (`LCXV13040`) are highlighted in yellow. These two mrp filetypes are fine for our purposes here; however if you want to download files associated with specific exposures, or any calibration files or intermediate files, you can select those you wish to download with the checkboxes in the file tree system (boxed in dashed-green)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### For this tutorial, we simply select Minimum Recommended Products at the top left. With this box checked, all of the folders representing individual exposures are no longer visible.\n",
+    "Check the box labelled \"HST\" to select all files included by the filters, and click the \"Download Selected Items\" button at the top right (dashed-black circle). This will bring up a small window asking you what format to download your files as. For datasets smaller than several Gigabytes, the `Zip` format will do fine. Click Download, and a pop-up window will try to open to download the files. If no download begins, make sure to enable this particular pop-up, or allow pop-ups on the MAST page.\n",
+    "\n",
+    "### There you go! Your files should now be downloaded as a compressed `Zip` folder. \n",
+    "If you need help uncompressing the `Zip`ped files, check out these links for: [Windows](https://support.microsoft.com/en-us/windows/zip-and-unzip-files-8d28fa72-f2f9-712f-67df-f80cf89fd4e5), [Mac](https://support.apple.com/guide/mac-help/zip-and-unzip-files-and-folders-on-mac-mchlp2528/mac). There are numerous ways to do this on Linux, however we have not vetted them."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id = astroqueryD></a>\n",
+    "# 2. The Python Package `astroquery.mast`\n",
+    "Another way to search for and download archived datasets is within `Python` using the module [`astroquery.mast`](https://astroquery.readthedocs.io/en/latest/mast/mast.html). We have already imported 2 of this module's key tools: `Observations` and `Mast`."
```

Comment:
It might be good to say here somewhere something like "the canonical source of information is the astroquery docs - look there for the most up-to-date instructions."  That's insurance against future changes that may make the advice this notebook makes in conflict with docs.